src/rgw/rgw_admin.cc

```diff
@@ -4247,10 +4247,26 @@ int main(int argc, const char **argv)
     bool need_cache = readonly_ops_list.find(opt_cmd) == readonly_ops_list.end();
     bool need_gc = (gc_ops_list.find(opt_cmd) != gc_ops_list.end()) && !bypass_gc;
 
+    std::string rgw_store = "rados";
+    const auto& config_store = g_conf().get_val<std::string>("rgw_backend_store");
+    #ifdef WITH_RADOSGW_DBSTORE
+    if (config_store == "dbstore") {
+      rgw_store = "dbstore";
+    }
+    #endif
+
+    #ifdef WITH_RADOSGW_MOTR
+    if (config_store == "motr") {
```

Comment:
should we use motrstore instead of motr? just to maintain the naming convention

Reply:
this code is enabled based on build flags, @andriytk I see this is done for radosgw as well any reason for this can we enable this always and let it be controlled in ceph.conf file only.